_shared/project/Makefile

```diff
@@ -135,16 +108,19 @@ $(call help,make {{ target }},"run the functional tests in Python {{ python_vers
 	@pyenv exec tox -qe py{{ python_version|pyformat(PyFormats.MAJOR_MINOR_FMT) }}-functests
 
 {% endfor %}
-.PHONY: sure
-$(call help,make sure,"make sure that the formatting$(comma) linting and tests all pass")
-sure: python
-sure:
+.PHONY: backend-sure
+backend-sure: python
+backend-sure:
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
 	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,coverage,functests'
 {% else %}
 	@pyenv exec tox --parallel -qe 'checkformatting,lint,tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage,functests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-functests'
 {% endif %}
 
+.PHONY: sure
+$(call help,make sure,"make sure that the formatting$(comma) linting and tests all pass")
+sure: backend-sure
```

Comment:
This change is also part of the `backend-sure` / getting `make sure` to run the backend stuff first